tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -277,7 +280,12 @@ static int spi_rx_half_end(struct device *dev, struct spi_config *spi_conf)
 	};
 	int ret;
 
-	LOG_INF("Start");
+	if (IS_ENABLED(CONFIG_SPI_STM32_DMA)) {
+		LOG_INF("Skip half end");
```

Comment:
why?

Reply:
This test makes sens when transferring with Fifo. In case of DMA, the transfer is started for a defined length in a complete burst. The half transfer is possibly detected for the 1st part. But it has not been enabled here. The second half is then detected when the complete transfer ends.

Reply:
ok